hwdb.d/60-seat.hwdb

```diff
@@ -28,6 +28,13 @@ fb:pci:v000018CAd00000020*
 fb:pci:v00001414d00005353*
  ID_TAG_MASTER_OF_SEAT=1
 
+# In some HyperV VMs, the video device only can be recognized by a vmbus id.
+# So we should add a vmbus id for HyperV Video device.
+#
+# This id is extracted from Linux kernel's hyperv.h
+fb:vmbus:02780ada77e3ac4a8e770558eb1073f8
```

Comment:
Let me confirm that vmbus id does not prefixed with `v`, unlike pci cases, right?

Reply:
![20240530145841](https://github.com/systemd/systemd/assets/32062239/43d53038-d224-4390-ab8c-92a8e9324a14)
I am working with the committer to investigate the issue with Hyper-V clients not displaying properly.
I used this patch to get the correct behavior. Without this patch, sddm and lightdm do not display the interface properly.

Reply:
yep, according the `udevadm info` output and the data in hwdb.d/20-vmbus-class.hwdb, it doesn't have a `v` prefix.

Reply:
Thanks.